tests/ara/test_variables.py

```diff
@@ -8,6 +8,7 @@
 
 @pytest.fixture(params=[
     RootInfo(name="root name", headers=["Root", "Name"], field="name"),
+    XOR(name="root name or sample_type", headers=["Root", "Info"], variables=[RootInfo(name="root name", headers=["Root", "Name"], field="name"), RootInfo(name="root name", headers=["Root", "Sample Type"], field="sample_type")]),
```

Comment:
This technically checks the functionality, but it sounds like a pretty implausible scenario.